downstream/modules/platform/proc-approving-the-imported-collection.adoc

```diff
@@ -2,45 +2,49 @@
+[NOTE] 
+====
+The collection is added to the "Published" repository regardless of its source.
+====
 
 Recommended collections depend on your use case. Ansible and Red Hat provide link:https://console.redhat.com/ansible/automation-hub[these collections.]
 
-== Custom Execution Environments
+== Custom [ExecEnvName]
+
+Use the ansible-builder program to create custom {ExecEnvShort} images. For disconnected environments, custom {ExecEnvShort} images can be built in the following ways:
 
-Use the ansible-builder program to create custom execution environment images. For disconnected environments, custom EE images can be built in the following ways:
+* Build an {ExecEnvShort} image on an internet-facing system and import it to the
+disconnected environment.
+* Build an {ExecEnvShort} image entirely on the disconnected environment with some modifications to the normal process of using ansible-builder.
+* Create a minimal base container image that includes all of the necessary modifications for a disconnected environment, then build custom {ExecEnvShort} images from the base container image.
 
-* Build an EE image on an internet-facing system and import it to the
-disconnected environment
-* Build an EE image entirely on the disconnected environment with some modifications to the normal process of using ansible-builder
-* Create a minimal base container image that includes all of the necessary modifications for a disconnected environment, then build custom EE images from the base container image
+=== Transferring Custom {ExecEnvShort} Images Across a Disconnected Boundary
 
-=== Transferring a Custom EE Images Across a Disconnected Boundary
+A custom {ExecEnvShort} image can be built on an internet-facing machine using the existing documentation. After an {ExecEnvShort} has been created it is available in the local podman image cache. You can then transfer the custom {ExecEnvShort} image across a disconnected boundary. 
 
-A custom execution environment image can be built on an internet-facing machine using the existing documentation.  Once an execution environment has been created it is available in the local podman image cache. You can then transfer the custom EE image across a disconnected boundary. To transfer the custom EE image across a disconnected boundary, first save the image:
+.Procedure
 
 . Save the image:
+
 ----
 $ podman image save localhost/custom-ee:latest | gzip -c custom-ee-latest.tar.gz
 ----
 
-Transfer the file across the disconnected boundary by using an existing mechanism such as sneakernet, one-way diode, etc..  After the image is available on the disconnected side, import it into the local podman cache, tag it, and push it to the disconnected hub:
-
+. Transfer the file across the disconnected boundary by using an existing mechanism such as sneakernet, one-way diode, etc. After the image is available on the disconnected side, import it into the local podman cache, tag it, and push it to the disconnected hub:
```

Comment:
Would remove etc. and say , or other. IBM style says not to use latin abbreviations.

Reply:
Changed it to "such as sneakernet or one-way diode." "Or other" may raise more questions rather than add clarification.

Reply:
sounds good!